tests/Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot

```diff
@@ -365,3 +366,11 @@ Verify Image Can Be Spawned
     Spawn Notebook With Arguments    ${retries}    ${image}    ${size}
     ...    ${spawner_timeout}    ${gpus}    ${refresh}    &{envs}
     End Web Test
+
+Verify Library Version Is Greater Than
+    [Arguments]     ${library}      ${target}
+    ${ver} =  Run Cell And Get Output   !pip show ${library} | grep Version: | awk '{split($0,a); print a[2]}' | awk '{split($0,b,"."); printf "%s.%s.%s", b[1], b[2], b[3]}'
+    ${comparison} =  GTE  ${ver}  ${target}
+    IF  ${comparison}==False
+        Run Keyword And Continue On Failure     FAIL    Library Version Is Smaller Than Expected
+    END
```

Comment:
I don't think you should do this in the keyword, IMHO it would be better to do this check in the TC if you need to do it at all

Reply:
I have did here just because whenever I'm using this keyword I don't have to write there also it will make the code redundant.